### PR TITLE
Fix isspace edge case

### PR DESCRIPTION
**Pull request by advikkabra**
An empty string should return `False` when the `isspace` method is called.

### CODE REVIEW

File: src/runtime/lpython_builtin.py [hunk 1]
```diff
@@ -963,6 +963,8 @@ def _lpython_str_isascii(s: str) -> bool:
     return True
 
 def _lpython_str_isspace(s:str) -> bool:
+    if (len(s) == 0):
+        return False
```
Comment (anutosh491) [inline src/runtime/lpython_builtin.py:967]: Looks good. We could skip the brackets though, `if len(s) == 0` should do !
Also add a test for the same.

**Comment by advikkabra** [comment]
Added the test, and also noticed a bug in the C++  implementation, so fixed the same.

**Comment by anutosh491** [comment]
> also noticed a bug in the C++ implementation

Ohh what's this ? Have you tested it out ? 
Remember to raise a separate PR for every other issue you are fixing. Helps us review and keep track of the change. Nothing wrong with atomic/small commits :)

**Comment by advikkabra** [comment]
It is the same edge case, just for constant empty strings as well. I will write a test for it as well.

**Review by certik — APPROVED** [review]
I think this is fine.

@anutosh491 can you please review & merge?

**Review by anutosh491 — APPROVED** [review]
Looks good, Merging !
